platform/ios/src/MGLMapView.mm

```diff
@@ -4118,6 +4114,27 @@ - (MGLCompactCalloutView *)calloutViewForAnnotation:(id <MGLAnnotation>)annotati
 /// Returns the rectangle that represents the annotation image of the annotation
 /// with the given tag. This rectangle is fitted to the image’s alignment rect
 /// and is appropriate for positioning a popover.
+/// If a shape annotation is visible but its centroid is not, and a default point is specified,
+/// the callout view is anchored to the default callout point.
+- (CGRect)positioningRectForAnnotation:(id <MGLAnnotation>)annotation withDefaultCalloutPoint:(CGPoint)calloutPoint
```

Comment:
Nit: in general, it’s unnecessary to use a preposition for a parameter after the first parameter (“with” in this case).